graphql/models/UserProfile.graphql

```diff
@@ -7,6 +7,7 @@ type UserProfile {
     occupation: String!
     address: String!
     state_of_origin: String!
+    source: Source!
```

Comment:
Rename this field to registeration_source so it is clear. Make the same change everywhere you previously used the "source" variable